src/main/java/frc/robot/Robot.java

```diff
@@ -96,6 +113,16 @@ public void teleopPeriodic() {
         } else {
             mIntakeMotor.set(ControlMode.PercentOutput, 0);
         }
+        if (m_XboxController.getAButton()) {
+            mSolenoidOne.get();
+            mSolenoidTwo.get();
+        } else if (m_XboxController.getBButton()) {
+            mSolenoidOne.toggle();
+            mSolenoidTwo.toggle();
+        } else {
+            mSolenoidOne.set(DoubleSolenoid.Value.kOff);
+            mSolenoidTwo.set(DoubleSolenoid.Value.kOff);
+        }
```

Comment:
If you want to have 2 different buttons to set pneumatics, you would want to use a button to make the pneumatics go forward and one to make them go back. and then turn them off in disabled as you have right now.